example/src/main/resources/public/index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Flagsmith Test</title>
+    <script src="https://code.jquery.com/jquery-3.6.0.min.js"
+            integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4="
+            crossorigin="anonymous"></script>
+</head>
+<body>
+    <p>
+        Hello <span id="identifier-name">World!</span>,
+        <button id="secret-button">Secret Button</button>
+    </p>
+    <p>
+        <form id="form">
+            Identify as a user:
+            <label for="identifier">Identifier: </label><input name="identifier" id="identifier"/>
+            <br />
+            With optional traits:
+            <label for="trait-key">Trait Key: </label><input name="trait-key" id="trait-key"/><br />
+            <label for="trait-value">Trait Value: </label><input name="trait-value" id="trait-value"/><br />
+
+            <button type="submit">Identify!</button>
+        </form>
+    </p>
+    <script type="text/javascript">
+        (function($) {
```

Comment:
Do we need to use JS here? Couldn't we use JSPs to render the HTML? This example is kind of redundant as we could, in this case, just use the JS SDK instead, right?

Reply:
Understood. I think using JSP would've required more configuration and dependencies, which is why this was done in this way. Let me use Spring Boot's thymeleaf to help render the page.